config/crd/bases/nnf.cray.hpe.com_nnfdatamovements.yaml

```diff
@@ -265,6 +279,11 @@ spec:
                       contain a valid percentage, then the value is unchanged from the previously parsed value.
                     format: int32
                     type: integer
+                  rate:
+                    type: string
+                  seconds:
+                    description: 'TODO: document these'
```

Comment:
What happened here?

Reply:
I forgot to rerun make generate && make manifests. Fixed now.